presto-spi/src/main/java/com/facebook/presto/spi/plan/ValuesNode.java

```diff
@@ -37,13 +37,15 @@
 {
     private final List<VariableReferenceExpression> outputVariables;
     private final List<List<RowExpression>> rows;
+    private final Optional<String> valuesNodeLabel;
```

Comment:
Could you add a comment to explain like "this is to add extra information for debugging purpose blab blab blah. It does not affect the execution of the value node blah blah blah"?